### ISSUE TITLE
Allow for other input formats when using the GAISSALabel plug-in

### ISSUE THREAD

**Issue by santidrj** [GAISSA-UPC/ML-GAISSALabel#98]
Currently the plug-in to generate the input file for the label generation only works for TensorFlow h5 models.
The plug-in should allow for other TensorFlow formats as well as PyTorch formats.

**Comment by santidrj** [comment]
Additionally it could support Hugging Face models. Although I don't know if it is possible
